packages/onr-core/src/mock/Mock.ts

```diff
@@ -0,0 +1,137 @@
+import { FactoryDefinition, ModelDefinition } from 'miragejs/-types';
+import { getDebugger } from '@onr/common';
+import { Serializer, Server } from 'miragejs';
+import { MockCallback } from './types';
+
+const debug = getDebugger('onr:core:mock');
+
+type IMock = {
+  resource: string;
+  model: ModelDefinition;
+  factory: FactoryDefinition;
+  serializer: Serializer;
+  seeds: any;
+  routes: any;
+  seedsCallback: MockCallback;
+  routesCallback: MockCallback;
+};
+
+export class Mock implements IMock {
+  public resource;
+  public model;
+  public factory;
+  public serializer;
+  public seeds;
+  public routes;
+  public seedsCallback;
+  public routesCallback;
+
+  constructor({
```

Comment:
recommend add static method for create server

Reply:
good idea, will create factory method for creating object.